### PR TITLE
Models directory can have sub directory

### PR DESCRIPTION
**Pull request by IkarosCoCo**
Models directory can have sub directory

### CODE REVIEW

File: lib/assets/models/index.js [hunk 1]
```diff
@@ -14,15 +14,25 @@ if (config.use_env_variable) {
...
+          var model = sequelize['import'](tmpPath);
+          db[model.name] = model;
+        } else {
+          init_db(tmpPath);
+        }
+      });
+  }
+}
+
+init_db(__dirname);
```
Comment (sdepold) [inline lib/assets/models/index.js]: Can we call it `readModels` or something like this?

File: lib/assets/models/index.js [hunk 2]
```diff
@@ -14,15 +14,25 @@ if (config.use_env_variable) {
...
-  .readdirSync(__dirname)
-  .filter(function(file) {
-    return (file.indexOf('.') !== 0) && (file !== basename) && (file.slice(-3) === '.js');
-  })
-  .forEach(function(file) {
-    var model = sequelize['import'](path.join(__dirname, file));
-    db[model.name] = model;
-  });
+function init_db(dir) {
+  if (fs.lstatSync(dir).isDirectory()) {
```
Comment (sdepold) [inline lib/assets/models/index.js]: Can we make it a guard like this:

```
function readModels (dir) {
  if(!fs.lstatsync(dir).isDirectory())) {
    return;
  } 

  /*the rest*/
}
```

File: lib/assets/models/index.js (hunk 1 above)
Comment (IkarosCoCo) [inline lib/assets/models/index.js]: yes,thanks for your time

File: lib/assets/models/index.js (hunk 2 above)
Comment (IkarosCoCo) [inline lib/assets/models/index.js]: yes,It is better

File: lib/assets/models/index.js [hunk 3]
```diff
@@ -14,15 +14,26 @@ if (config.use_env_variable) {
...
+  if(!fs.lstatsync(dir).isDirectory()) {
+    return;
+  }
+    fs.readdirSync(dir)
+      .filter(function (file) {
+        return file !== basename;
+      })
+      .forEach(function (file) {
+        var tmpPath = path.join(dir, file);
+        if ((!fs.lstatSync(tmpPath).isDirectory()) && (tmpPath.slice(-3) === '.js')) {
```
Comment (Americas) [inline lib/assets/models/index.js]: these conditions should be separated, you don't want it to call readModels if the file isn't ending with '.js'

You also removed the condition that does not read hidden files.

**Comment by sushantdhiman** [comment]
Closing (PR Cleanup), Open new PR if you wish to continue your work

**Pull request closed without merging**